uint32 id
float32 temp
float32 pot_imp
float32 plant_imp